Framework/PythonInterface/plugins/algorithms/SCDAdjustSampleNorm.py

```diff
@@ -0,0 +1,246 @@
+# Mantid Repository : https://github.com/mantidproject/mantid
+#
+# Copyright &copy; 2020 ISIS Rutherford Appleton Laboratory UKRI,
+#   NScD Oak Ridge National Laboratory, European Spallation Source,
+#   Institut Laue - Langevin & CSNS, Institute of High Energy Physics, CAS
+# SPDX - License - Identifier: GPL - 3.0 +
+from mantid.api import AlgorithmFactory, FileAction, FileProperty, IMDHistoWorkspaceProperty, PythonAlgorithm, \
+    PropertyMode, MultipleFileProperty, WorkspaceProperty
+from mantid.kernel import Direction, EnabledWhenProperty, PropertyCriterion, V3D, FloatArrayProperty, \
+    FloatArrayLengthValidator
+from mantid.simpleapi import ConvertHFIRSCDtoMDE, ConvertWANDSCDtoQ, DeleteWorkspace, DeleteWorkspaces, DivideMD, \
+    Load, MergeMD, ReplicateMD, mtd
+import os
+
+
+class SCDAdjustSampleNorm(PythonAlgorithm):
```

Comment:
This algorithm name is too general, it suggests that it can be used for any single-crystal instrument when it will only work for DEMAND.

Maybe include DEMAND or HB3A in the name.

Reply:
In addition to changing the name, I think this can be converted to a [WorkflowAlgorithm](https://docs.mantidproject.org/nightly/concepts/WorkflowAlgorithm.html) and moved to `Framework/PythonInterface/plugins/algorithms/WorkflowAlgorithms/`